src/main/java/Deadline.java

```diff
@@ -0,0 +1,24 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Deadline extends Task {
+
+    protected LocalDate by;
```

Comment:
Perhaps this access modifier could be private since I do not see it being accessed outside of this class? Or is there any reason why you chose to use "protected"?

Reply:
I chose protected because I think both protected and private may work, and protected may help if any access is needed in the future code implementation HAHA. Thank you for your suggestion!